.github/workflows/main.yml

```diff
@@ -7,16 +7,15 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby: [2.7, 3.0, 3.1]
+        ruby: [2.7, 3.0, 3.1, 3.2, 3.3]
     name: Ruby ${{ matrix.ruby }}
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v4
     - name: Set up Ruby
       uses: ruby/setup-ruby@v1
       with:
         ruby-version: ${{ matrix.ruby }}
     - name: Run the default task
       run: |
-        gem install bundler -v 2.2.3
```

Comment:
nit change: We don't need to specify the bundler version

> If there is a Gemfile.lock file (or $BUNDLE_GEMFILE.lock or gems.locked) with a BUNDLED WITH section, that version of Bundler will be installed and used.

https://github.com/ruby/setup-ruby?tab=readme-ov-file#bundler